packages/manager/apps/telecom/src/components/telecom/telephony/timeCondition/condition/edit/telephony-time-condition-condition-edit.controller.js

```diff
@@ -196,11 +198,12 @@ angular.module('managerApp').controller('voipTimeConditionConditionCtrl', functi
     refreshTime();
   };
 
-  self.onTimeToChange = function onTimeToChange() {
+  self.onTimeToChange = function onTimeToChange(modelValue) {
+    self.model.timeTo = modelValue;
     const timeToMoment = moment(self.model.timeTo);
-    const isMidnight = timeToMoment.get('hour') === 0 && timeToMoment.get('minute') === 0;
-    if (!isMidnight && self.model.timeFrom >= self.model.timeTo) {
-      self.model.timeFrom = moment(self.model.timeTo).subtract(15, 'minute').toDate();
+    const isEndMidnight = timeToMoment.get('hour') === 0 && timeToMoment.get('minute') === 0;
+    if (!isEndMidnight && self.model.timeFrom >= self.model.timeTo) {
+      self.model.timeTo = moment(self.model.timeFrom).add(15, 'minute').toDate();
```

Comment:
It seems to keep the same behaviour but any reason why such a change ?

Reply:
My mistake. Ahh i will revert this. Thanks